src/config/2021.json

```diff
@@ -454,7 +462,16 @@
       "teams": [
         "designers"
       ]
-    },"nithanaroy": {
+    },
+    "Navaneeth-akam": {
```

Comment:
FYI @Navaneeth-akam adding your contributor info. Open a new PR to make any changes.

Reply:
This looks ok. If you need any more info, let me know